pages/team.php

```diff
@@ -363,6 +397,33 @@ function team() {
                     $system->query("UPDATE `teams` SET `mission_id`='{$mission_id}' WHERE `team_id`='{$player->team->id}' LIMIT 1");
                     if($system->db_last_affected_rows) {
                         $system->message("Mission started!");
+                        // Create notification
+                        $result = $system->query("SELECT `name` FROM `missions` WHERE `mission_id` = {$mission_id}");
+                        $mission_name = $system->db_fetch($result)['name'];
+                        require_once __DIR__ . '/../classes/notification/NotificationManager.php';
```

Comment:
Per convention, let's move this require up to the top of the page (difference with this vs layout/templates is that requiring the NotificationManager does not trigger any effects, just makes the class available)